configurator/src/Configurator/Vendor/Keycloak/KeycloakManager.php

```diff
@@ -319,13 +319,15 @@ public function updateClientByClientId(string $clientId, array $data = []):void
     {
         $client = $this->getClientByClientId($clientId);
 
-        $this->getAuthenticatedClient()
+        if (null !== $client) {
```

Comment:
`updateClientByClientId` should fail if client does not exist.
`\App\Configurator\Vendor\Keycloak\KeycloakManager::createClient` actually supports existing client!

Reply:
Still TODO